components/form/style/rtl.less

```diff
@@ -32,6 +32,11 @@
           margin-left: 4px;
         }
       }
+      &::after {
+        .@{form-prefix-cls}-rtl & {
+          margin: 0 @form-item-label-colon-margin-left 0 @form-item-label-colon-margin-right;
```

Comment:
According to the specifications for **margin**, I believe `@form-item-label-colon-margin-left` and `@form-item-label-colon-margin-right` should be interchanged.